src/main/java/transportservice/ExtensionSettings.java

```diff
@@ -0,0 +1,38 @@
+package transportservice;
+
+public class ExtensionSettings {
+
+    private String nodename;
```

Comment:
nit: Lets start moving away from the node. 
```suggestion
    private String name;
```